src/batterystatus.cpp

```diff
@@ -44,10 +44,24 @@ const QString MceSettingsChargingMode = QStringLiteral("/system/osso/dsm/chargin
 const QString MceSettingsChargingLimitEnable = QStringLiteral("/system/osso/dsm/charging/limit_enable");
 const QString MceSettingsChargingLimitDisable = QStringLiteral("/system/osso/dsm/charging/limit_disable");
 
+const QString MceForcedChargingEnabled = QStringLiteral(MCE_FORCED_CHARGING_ENABLED);
+const QString MceForcedChargingDisabled = QStringLiteral(MCE_FORCED_CHARGING_DISABLED);
+
 const int MceChargingModeDisable = 0;
 const int MceChargingModeEnable = 1;
 const int MceChargingModeApplyThresholds = 2;
 const int MceChargingModeApplyThresholdsAfterFull = 3;
+
+bool chargingOverrideFromDBus(QString value)
```

Comment:
I'd follow the convention and pass `value` by const reference

Reply:
done